Turning project name from snake_case into kebab-case

Hi Dioxus nice people! :blush: 

Using Dioxus CLI v.`0.1.5`, there are a couple of small issues with the creation part:
1. Snake case names (like `some_project`) become kebab case names (like `some-project`).
2. Probably as a side effect, the project name in `Cargo.toml` also gets specified using kebab case.

```
$ dioxus create router_test
[INFO] 🔧 Start to create a new project 'router_test'.
[WARN] Tool is not installed: cargo-generate, try to install it.
    Updating crates.io index
  Downloaded cargo-generate v0.16.0
   ...
   Compiling cargo-generate v0.16.0
    Finished release [optimized] target(s) in 2m 35s
  Installing /home/dxps/.cargo/bin/cargo-generate
   Installed package `cargo-generate v0.16.0` (executable `cargo-generate`)
[1/9]   Done: .gitignore
[2/9]   Done: Cargo.toml
[3/9]   Done: Dioxus.toml
[4/9]   Done: LICENSE
[5/9]   Done: README.md
[6/9]   Done: public/.gitkeep
[7/9]   Done: public
[8/9]   Done: src/main.rs
[9/9]   Done: src
$ 
```
Ofc, once `cargo-generate` is already installed, things look kinda similar:
```shell
❯ dioxus create some_project
[INFO] 🔧 Start to create a new project 'some_project'.
[1/9]   Done: .gitignore
[2/9]   Done: Cargo.toml
[3/9]   Done: Dioxus.toml
[4/9]   Done: LICENSE
[5/9]   Done: README.md
[7/9]   Done: public
[8/9]   Done: src/main.rs
[9/9]   Done: src
[ERROR] 🚫 Create project failed: I/O Error: No such file or directory (os error 2)
❯
```
As before, a `some-project` folder is created.

--- a/src/cli/create/mod.rs
+++ b/src/cli/create/mod.rs
@@ -53,6 +53,7 @@ impl Create {
             .arg(&self.template)
             .arg("--name")
             .arg(&self.name)
+            .arg("--force")
             .stdout(Stdio::piped())
             .stderr(Stdio::inherit())
             .output()?;

--- a/src/main.rs
+++ b/src/main.rs
@@ -7,7 +7,7 @@ async fn main() -> Result<()> {
     let args = Cli::parse();
     set_up_logging();
 
-    let dioxus_config = DioxusConfig::load()?;
+    let dioxus_config = DioxusConfig::load().unwrap_or(DioxusConfig::default());
 
     let plugin_state = PluginManager::init(dioxus_config.plugin);
 

